### ISSUE TITLE
"Spark Agency" is marked as rotated when choosing an ID for a new deck & when editing the deck

### ISSUE THREAD

**Issue by Neuvost** [Null-Signal-Games/netrunnerdb#535]
"Spark Agency: Worldswide Reach" was in SysCore19—which has rotated—but it's also in Data & Destiny—which has not. I looked through the rest of SysCore19, and I don't think this comes up with any other cards. 

¡Thanks for being beautiful!

**Comment by NoahTheDuke** [comment]
Thanks for the report!

**Comment by plural** [comment]
oh.  hahahaha

i think this is because the *latest printing* is actually in a rotated
pack.  grumble grumble grumble.

**Comment by plural** [comment]
i just tried this and it correctly shows the Data & Destiny printing, not the SC 2019 version and i don't see a rotated icon.

do you have a screen capture or any more reproduction details?

**Comment by Neuvost** [comment]
Screen caps:
https://imgur.com/a/aPlwJZN

**Comment by TradonNetrunner** [comment]
I just yesterday stumbled onto the same thing with paragon (rotated with system core 2019, but still viable via reign&reverie)

I just now checked it for reproducability and it seems to be gone.
I had it yesterday evening after not buildings decks since before gateway, so I had some old lists with paragon and those showed it as "rotated" and after updateting the "collection" to standard without the system core 2019 yesterday, paragon wasn't listed in the crim hardware anymore at all and the decklist marked it as "rotated".
Today it was shown as the R&R-version correctely in the list and in the decklist again.

I just clicked around a bit further and managed to reproduce it:
1. I looked up some published decklist from APAC - so before rotation which included Paragon.
2. I Copied this decklist into my decks and there opened it for editing.
3. I clicked the button for "Deck contains 2 rotated cards with new versions - click to update" and afterwards it still shows Paragon as the rotated-system core 2019-version in the decklist.
![01](https://user-images.githubusercontent.com/85844733/121818294-87be1f00-cc86-11eb-9c57-b6d23b083e56.JPG)
![02](https://user-images.githubusercontent.com/85844733/121818299-8ee52d00-cc86-11eb-890d-d9c2e801ae6f.jpg)
